ksp_plugin_adapter/config_node_parsers.cs

```diff
@@ -89,10 +91,41 @@ public static ConfigurationFixedStepParameters
         angular_frequency    =
             node?.GetAtMostOneValue("angular_frequency")
                 ??  (body.angularV.ToString() + " rad/s"),
+        reference_radius     =
+            node?.GetAtMostOneValue("reference_radius"),
         j2                   =
             node?.GetAtMostOneValue("j2"),
-        reference_radius     =
-            node?.GetAtMostOneValue("reference_radius")};
+        geopotential            =
+            GetBodyGeopotentialElements(node)};
+  }
+
+  private static BodyGeopotentialElement[] GetBodyGeopotentialElements(
```

Comment:
Hmm, but then the pointer is no longer null in case of an empty array.  That's probably fine, although the generated code has `?.` and `??` to deal with the null case.

Done.